Gemfile

```diff
@@ -45,7 +45,9 @@ gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
 gem "bootsnap", require: false
```

Comment:
We may also want to dump sprockets (line 10) for propshaft, which works hand-in-hand with cssbundling-rails.